src/index.html

```diff
@@ -21,7 +24,389 @@
+        </div>
+
+        <div class="product-card__product-code">Product code: 195434</div>
+
+        <div class="product-card__reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <div class="product-card__reviews-count">Reviews: 5</div>
+        </div>
+
+        <div class="product-card__price-block">
+          <p class="product-card__price-text">Price:</p>
+
+          <p class="product-card__price">$2,199</p>
+        </div>
+
+        <button class="product-card__button">Buy</button>
+      </div>
+
+      <div class="product-card">
+        <div class="product-card__block">
+          <img
+            src="./images/imac.jpeg"
+            alt="product card image"
+            class="product-card__image"
+          />
+        </div>
+
+        <div class="product-card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+
+        <div class="product-card__product-code">Product code: 195434</div>
+
+        <div class="product-card__reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <div class="product-card__reviews-count">Reviews: 5</div>
+        </div>
+
+        <div class="product-card__price-block">
+          <p class="product-card__price-text">Price:</p>
+
+          <p class="product-card__price">$2,199</p>
+        </div>
+
+        <button class="product-card__button">Buy</button>
+      </div>
+
+      <div class="product-card">
+        <div class="product-card__block">
+          <img
+            src="./images/imac.jpeg"
+            alt="product card image"
+            class="product-card__image"
+          />
+        </div>
+
+        <div class="product-card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+
+        <div class="product-card__product-code">Product code: 195434</div>
+
+        <div class="product-card__reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <div class="product-card__reviews-count">Reviews: 5</div>
+        </div>
+
+        <div class="product-card__price-block">
+          <p class="product-card__price-text">Price:</p>
+
+          <p class="product-card__price">$2,199</p>
+        </div>
+
+        <button class="product-card__button">Buy</button>
+      </div>
```

Comment:
The repeated `product-card` blocks should be generated using a loop in your template engine or front-end framework, rather than being hard-coded. If this is a static HTML file, consider this as a suggestion for when you move to a dynamic solution.